webcompat/templates/form.html

```diff
@@ -130,7 +130,7 @@ <h1 class="headline-1">Report Site Issue</h1>
         <button
           class="button button-secondary {{ 'js-Button' if anon_reporting_enabled else '' }}"
           id="submitanon"
-          type="button"
+          type={{ 'submit' if anon_reporting_enabled else 'button' }}
```

Comment:
I don't think I understand this part of the code here. if `anon_reporting_enabled` is `False` should all the markup go away instead of just changing the type?

Reply:
Yeah, my bad. This is a confusing ride-along commit in the context of this PR. I just noticed I forgot to include it as part of #3132

Changing the button `type` is just an extra-careful measure, it will already be `disabled`, and the text will say it's disabled, and the backend will reject any issues that somehow slip through. If you feel strongly about a different proposal, can you file a follow-up issue? I don't have strong feelings about the way I've done it here.